app/(gcforms)/[locale]/(form administration)/forms/[[...path]]/page.tsx

```diff
@@ -38,7 +52,36 @@ export default async function Page({
       redirect: true,
     });
 
-    const { t } = await serverTranslation("my-forms");
+    const {
+      t,
+      i18n: { language },
+    } = await serverTranslation("my-forms");
+
```

Comment:
I added the below comment because I know in the future, if I were to see this, I'd be very tempted to move this call into the Cards component. It just feels cleaner to me. Though I couldn't find a way to stop the component being cached when navigating back to the page.